README.md

```diff
@@ -149,9 +146,15 @@ The following global settings can be used under the "no-jquery" property to conf
 * [no-jquery/no-unique](docs/no-unique.md)
 * [no-jquery/no-unload-shorthand](docs/no-unload-shorthand.md)
 * [no-jquery/no-val](docs/no-val.md)
+* [no-jquery/no-visibility](docs/no-visibility.md)
 * [no-jquery/no-when](docs/no-when.md)
 * [no-jquery/no-wrap](docs/no-wrap.md)
 
+### Deprecated
```

Comment:
Should we explain for deprecated rules what people should use instead in the main docs, not just the individual ones?

Reply:
That could get messy. Ideally ESLint would output it during the run, but it doesn't.